common/src/main/java/com/wynntils/features/inventory/CustomBankQuickJumpsFeature.java

```diff
@@ -106,6 +87,7 @@ public void onScreenInit(ScreenInitEvent e) {
 
     private void getCustomJumpDestinations() {
         String configDestinations;
+        SearchableContainerType currentContainer = Models.Bank.getCurrentContainer();
```

Comment:
you don't use this?

Reply:
It's used in the switch and passing into `parseStringToDestinations`?

Reply:
oh right. Sorry.

Reply:
Although, maybe you could just call it there?